web/models.py

```
@@ -2787,3 +2787,105 @@ class ScheduledPost(models.Model):
+
+class LearningMap(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="learning_maps")
+    title = models.CharField(max_length=100)
+    description = models.TextField(blank=True)
+    slug = models.SlugField(max_length=150, unique=True, blank=True)
+    public = models.BooleanField(default=False, help_text="If enabled, this map can be viewed by anyone with the link")
+    share_token = models.CharField(max_length=32, unique=True, blank=True, editable=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def save(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `save`

Add return type annotation: `None`

(ANN201)